components/HeaderSingle/HeaderSingle.js

```diff
@@ -0,0 +1,22 @@
+import Link from 'next/link';
+
+import { SearchBar } from '../SearchBar';
+import { Navigation } from '../Navigation';
+
+export const HeaderSingle = () => (
+  <header className="w-full h-20">
+    <div className="w-full bg-light-50 fixed z-50 top-0 transition drop-shadow-md">
+      <div className="max-w-screen-xl mx-auto py-2 px-4 md:p-3 flex relative lg:items-center">
+        <Link href="/">
+          <a className="font-yrsa font-bold uppercase text-4xl leading-none pr-8 text-black  hover:text-opacity-80 transition duration-200">
+            Hiswiki
+          </a>
+        </Link>
+
+        <SearchBar isScrolledDown="true" />
```

Comment:
Well, there are two things I can say about this line.

First of all, be very very very careful what you have just written here. You are sending inside a string. Not a boolean. That luckily works at the moment. But try to pass in `isScrolledDown="false"`. It would behave exactly the same way. What you wanted is `isScrolledDown={true}`.

You might get confused by the fact an eslint could shorten it for you to `isScrolledDown`. But that is equivalent. And that is the second thing I wanted to point out. I prefer the shorter version for boolean constants in JSX.
There is an eslint rule for it


```   "react/jsx-boolean-value": 2, // <Component someFlag={true} /> results to <Componet someFlag />```

Reply:
I was not confused. I was wrong and I wrote nonsense :D as usually